misc/helm-charts/operator/values.yaml

```diff
@@ -227,17 +227,22 @@ serviceAccount:
 # Observability settings (disabled in this case)
 observability:
   enabled: false
+  podMetrics:
+    # Whether to enable the pod metrics scraper which populates the
+    # Environment Overview Monitoring tab in the web console (requires
+    # metrics-server to be installed)
+    enabled: false
   prometheus:
     # Whether to enable Prometheus integration for monitoring (disabled here)
     enabled: false
 
 # Network policies configuration
 networkPolicies:
   # Whether to enable network policies for securing communication between pods
-  enabled: false
+  enabled: true
```

Comment:
Did we mean to re-enable these? I thought we wanted them disabled by default.

Reply:
whoops, i'll fix (they need to be enabled for testing with the cloud kind cluster, so i keep doing this by accident)